ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -28,11 +28,11 @@ base_images:
     namespace: ci
     tag: "4.15"
   tools:
-    name: "4.15"
+    name: "4.16"
     namespace: ocp
     tag: tools
   upi-installer:
-    name: "4.15"
+    name: "4.16"
```

Comment:
As this is 4.15 upgrade to 4.16, we are installing on 4.15, IMO, it should be 4.15 here.

Reply:
agree.

Reply:
done